Update the "leaving space for others" arc

ctaymor opened the pull request:
* Move "actively leaves space for others to share their ideas to P2"
because P1s are usually still developing enough ideas to be comfortable
sharing them at all. P2 is still the "do it yourself" level of
contribution, so I think this skill makes sense there.
* Add a P3 skill "Actively works to ensure quieter teammates have room
to share and get recognition for their ideas". P3 is doing things that
have a whole team impact. One can start in this arc by thinking about
the amount of space you take up in the room (P2), and then start
noticing and influencing the amount of space others take up in the room.
* Add a P4 skill "Coaches pivots on how to facilitate conversations and
meetings that allow other quieter teammates to share and get recognition
for their ideas". Noticing how much space one takes up in the room, and
how much space others take up, and balancing that, isn't an easy skill
to develop. Coaching other pivots on how to do this skill would have a
big impact on the team, by enabling other pivots.

Followup PR from https://github.com/pivotal-cf/areas-of-contribution/issues/32

Commit:
Update the "leaving space for others" arc

* Move "actively leaves space for others to share their ideas to P2"
because P1s are usually still developing enough ideas to be comfortable
sharing them at all. P2 is still the "do it yourself" level of
contribution, so I think this skill makes sense there.
* Add a P3 skill "Actively works to ensure quieter teammates have room
to share and get recognition for their ideas". P3 is doing things that
have a whole team impact. One can start in this arc by thinking about
the amount of space you take up in the room (P2), and then start
noticing and influencing the amount of space others take up in the room.
* Add a P4 skill "Coaches pivots on how to facilitate conversations and
meetings that allow other quieter teammates to share and get recognition
for their ideas". Noticing how much space one takes up in the room, and
how much space others take up, and balancing that, isn't an easy skill
to develop. Coaching other pivots on how to do this skill would have a
big impact on the team, by enabling other pivots.

Relates to #issue-32

rosenhouse commented:
Labeling this issue with `arc` so that when we get around to it, we can go back and annotate these as belonging to a single arc.

rnandi commented:
I really like the way the P3 and P4 versions of this arc are framed. How about framing the P2 version as: "Shares thoughts in a way that fosters further conversation and leaves space for others "

ctaymor commented:
@rnandi I think that's a big improvement for the P2 version. It really captures that it's not about being quiet, but about leaving space for others when you **are** sharing ideas.

rosenhouse commented:
@ctaymor would you like to make that change to this PR?  We'd be happy to merge it after that!

Commit:
Update the P2 leaving space for others skill

based on @rnandi's suggested improvement, which captures that this
skill is not about being quiet, but about how you leave space for
others when you **are** sharing ideas.

ctaymor commented:
Sure. I updated the PR with @rnandi's suggestion, and also rebased over the latest master.
